packages/dev/storybook-builder-parcel/index.js

```diff
@@ -62,6 +62,8 @@ exports.build = async function ({options}) {
 exports.corePresets = [];
 exports.previewPresets = [];
 
+exports.bail = async function (e) {};
```

Comment:
What is this for?

Reply:
ah, that slipped by me
shouldln't hurt anything, but it's required when we go to upgrade to sb7